plugins/scanners/scancode/src/test/assets/scancode-32.0.8_spdx-expression-parse_no-license-references.json

```diff
@@ -0,0 +1,823 @@
+      "percentage_of_license_text": 11.25,
+      "copyrights": [],
+      "holders": [],
+      "authors": [],
+      "files_count": 0,
+      "dirs_count": 0,
+      "size_count": 0,
+      "scan_errors": []
+    },
+    {
+      "path": ".github",
+      "type": "directory",
+      "name": ".github",
+      "base_name": ".github",
+      "extension": "",
+      "size": 0,
+      "date": null,
+      "sha1": null,
+      "md5": null,
+      "sha256": null,
+      "mime_type": null,
+      "file_type": null,
+      "programming_language": null,
+      "is_binary": false,
+      "is_text": false,
+      "is_archive": false,
+      "is_media": false,
+      "is_source": false,
+      "is_script": false,
+      "detected_license_expression": null,
+      "detected_license_expression_spdx": null,
+      "license_detections": [],
+      "license_clues": [],
+      "percentage_of_license_text": 0,
+      "copyrights": [],
+      "holders": [],
+      "authors": [],
+      "files_count": 1,
+      "dirs_count": 1,
+      "size_count": 371,
+      "scan_errors": []
+    },
+    {
+      "path": ".github/workflows",
+      "type": "directory",
+      "name": "workflows",
+      "base_name": "workflows",
+      "extension": "",
+      "size": 0,
+      "date": null,
+      "sha1": null,
+      "md5": null,
+      "sha256": null,
+      "mime_type": null,
+      "file_type": null,
+      "programming_language": null,
+      "is_binary": false,
+      "is_text": false,
+      "is_archive": false,
+      "is_media": false,
+      "is_source": false,
+      "is_script": false,
+      "detected_license_expression": null,
+      "detected_license_expression_spdx": null,
+      "license_detections": [],
+      "license_clues": [],
+      "percentage_of_license_text": 0,
+      "copyrights": [],
+      "holders": [],
+      "authors": [],
+      "files_count": 1,
+      "dirs_count": 0,
+      "size_count": 371,
+      "scan_errors": []
+    },
+    {
+      "path": ".github/workflows/ci.yml",
+      "type": "file",
+      "name": "ci.yml",
+      "base_name": "ci",
+      "extension": ".yml",
+      "size": 371,
+      "date": "2023-12-08",
+      "sha1": "8c07b83a6668af392339d64118b31e99b87aac2c",
+      "md5": "e6a9fe1a11e629e5fc9b2c98be054059",
+      "sha256": "915895e3d84158dd736dac6a5aaf55ea62bc9441d6e2b35875e8b5416652656c",
+      "mime_type": "text/plain",
+      "file_type": "ASCII text",
+      "programming_language": null,
+      "is_binary": false,
+      "is_text": true,
+      "is_archive": false,
+      "is_media": false,
+      "is_source": false,
+      "is_script": false,
+      "detected_license_expression": null,
+      "detected_license_expression_spdx": null,
+      "license_detections": [],
+      "license_clues": [],
+      "percentage_of_license_text": 0,
+      "copyrights": [],
+      "holders": [],
+      "authors": [],
+      "files_count": 0,
+      "dirs_count": 0,
+      "size_count": 0,
+      "scan_errors": []
+    }
+  ]
+}
```

Comment:
Nit: Missing newline at EOF.

Reply:
True. I took the file as-is from ScanCode, and it's inconsistent for our JSON assets anyway, so I'll align this another way 😸

Reply:
As there was another fixup I had to do, I've actually decided to align this now, too.